src/codegen/ast_interpreter.cpp

```diff
@@ -609,6 +630,21 @@ Value ASTInterpreter::visit_invoke(AST_Invoke* node) {
         v = visit_stmt(node->stmt);
         next_block = node->normal_dest;
     } catch (ExcInfo e) {
+
+        if (threading::ThreadStateInternal::getUnwindState() == UNWIND_STATE_NORMAL) {
```

Comment:
to answer my own question, no they aren't.  but using this form seems to work.  I dislike the idea slightly of having multiple ways to check if a raise was a re-raise depending on what context we're in, but this particular instance doesn't really pain me too much, as the only way for there to be a re-raise is for the `node->stmt` to have caused it.